test/Exporter/Test/Writer/FormatedBoolWriterTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Exporter\Test\Source;
+
+use Exporter\Writer\FormatedBoolWriter;
+use Exporter\Writer\XlsWriter;
+
+class FormatedBoolWriterTest extends \PHPUnit_Framework_TestCase
+{
+    protected $writer;
+
+    protected $sampleWriter;
+
+    protected $trueLabel;
+
+    protected $falseLabel;
+
+    protected $filename;
+
+    public function setUp()
+    {
+        $this->filename = 'formatedbool.xls';
```

Comment:
no, just use a mock with expected method call and check calling arguments: see https://phpunit.de/manual/current/en/test-doubles.html#test-doubles.mock-objects.examples.with-consecutive.php